lib/actions-oclif/env/rename.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * @license
+ * Copyright 2016-2019 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Command, flags } from '@oclif/command';
+import { stripIndent } from 'common-tags';
+
+import * as cf from '../../utils/common-flags';
+import { CommandHelp } from '../../utils/oclif-utils';
+
+type IArg<T> = import('@oclif/parser').args.IArg<T>;
+
+interface FlagsDef {
+	device: boolean;
+	help: void;
+}
+
+interface ArgsDef {
+	id: number;
+	value: string;
+}
+
+export default class EnvRenameCmd extends Command {
+	public static description = stripIndent`
+		Change the value of an environment variable for an app or device.
```

Comment:
I wonder how we ended up with a command called `rename` that changes the value instead of the name :/ Do you plan a major CLI release soon @pdcastro? Definitely worth to rename the command to `set` or something similar when we do. In fact, you could do this here properly and have a shim for the `rename` command with a deprecation warning. Does oclif make this kind of thing difficult?

Reply:
Thanks - no major release currently planned, but I created a document (as suggested by Lucian) to keep track of suggested breaking changes for a future major release: https://docs.google.com/document/d/1uHE-_LLu5gJUQZICQJtlo76EQCm90z0CMAwpkfw6s8w/edit?usp=sharing

However I think that you and Lucian have only made points about the `rename` command because I noticed and highlighted its oddity in the first place. We have a substantial backlog of issues/bugs that have been reported by internal and external users (things that are "truly bothering" them), so I would give priority to addressing those issues.